### DISCUSSION TITLE
Add a Save button to Layout Editor

### DISCUSSION THREAD

**Discussion by yvvt0379** [tyron12233/CodeAssist#310] [Ideas]
(Since there's no “Feature Request” tab, I have to choose “No template” lol)

When I manipulate the layout in Layout Editor, changes are not applied to the layout xml file, so I have to edit code, which is inconvenient. I hope that a Save button can be added which applies the changes to the layout xml when clicked.

**Comment by alluhemanth** [comment]
when you click back button it shows a dialog to save

**Comment by yvvt0379** [comment]
However, when I save it again, the file restores back as if no changes has been made.
